src/test/input_byte_size_test.cc

```diff
@@ -629,7 +643,8 @@ TEST_F(InputByteSizeTest, StringCountMismatchGPU)
   // Create an inference request
   FAIL_TEST_IF_ERR(
       TRITONSERVER_InferenceRequestNew(
-          &irequest_, server_, "simple_identity", -1 /* model_version */),
+          &irequest_, server_, "savedmodel_zero_1_object",
```

Comment:
What was the purpose of changing the model used here?

Reply:
I changed non-string tests to use `qa_identity_model_repository/savedmodel_zero_1_float32` because it has fewer inputs than `docs/examples/model_repository/simple`. Updated `docs/examples/model_repository/simple_identity` to `qa_identity_model_repository/savedmodel_zero_1_object` just to unify their source location.